lib/kitchen/directions/bake_chapter_content_module.rb

```diff
@@ -0,0 +1,13 @@
+module Kitchen
+  module Directions
+    module BakeChapterContentModule
```

Comment:
The thing that is being baked here is the abstract in a page.  So I think the name of this class should reflect that somewhat.  Even just `BakeAbstracts` or `BakePageAbstracts`.  Note also that the recipes already iterate through chapters, so if you wanted to leave this as working on a chapter, your method could be `def self.v1(chapter:)` and then call this direction from inside a chapter loop in the recipe.  This latter part is your call, whatever makes the most sense.  

Are there ever abstracts that are not in chapters?  If so `book.search('[data-type="abstract"]')` would be better.  There may not be tho.

Reply:
There are not abstracts (at least, not ones that require this insertion) outside of chapters, at least not in chemistry.